package.json

```diff
@@ -82,7 +83,7 @@
     "pinst": "^2.1.6",
     "prettier": "^2.2.1",
     "semantic-release": "^17.3.0",
-    "stylelint": "^14.0.1",
+    "stylelint": "^15.0.0",
```

Comment:
Upgrading `v15` yields dozens of type errors
> TypeError: node.rangeBy is not a function
    at Object.report (/Users/andy/work/stylelint-declaration-strict-value/node_modules/stylelint/lib/utils/report.js:35:34)

- [ ] https://github.com/stylelint/stylelint/issues/5756
- [ ] https://github.com/stylelint/stylelint/issues/5766

Reply:
Seems that some patches are being prepared 🤔 
https://github.com/stylelint/stylelint/issues/6647

Reply:
Does [15.1.0](https://github.com/stylelint/stylelint/releases/tag/15.1.0) help any?

Reply:
No.
The issue is the old tape test utils, they are deprecate and archived.

The jest test utils do not support testing a plugins options validation.

I will finish it as soon as I free time available.